Src/Sarif.PatternMatcher.Cli/Program.cs

```diff
@@ -18,11 +18,23 @@ internal static class Program
         [ThreadStatic]
         internal static IFileSystem FileSystem;
 
+        [ThreadStatic]
+        internal static Exception RuntimeException;
+
         internal static int Main(string[] args)
         {
             Console.OutputEncoding = Encoding.UTF8;
 
-            args = EntryPointUtilities.GenerateArguments(args, FileSystem, new EnvironmentVariables());
+            try
+            {
+                args = EntryPointUtilities.GenerateArguments(args, FileSystem ?? new FileSystem(), new EnvironmentVariables());
```

Comment:
>GenerateArguments [](http://example.com/codeflow?start=43&length=17)

This code path falls entirely out of our exception handling infrastructure. That's ok, in the sense that we would fail catastrophically in any case, but we need some machinery to capture the exception for test purposes.